### ISSUE TITLE
Building fixes/32 packages for Debian 10 or 11 fails to clone from GitHub

### ISSUE THREAD

**Issue by Tinlegs** [MythTV/mythtv#651]
ant@ripley:~/build32$ git clone https://github.com/MythTV/packaging.git -b fixes/32
Cloning into 'packaging'...
remote: Enumerating objects: 15473, done.
remote: Counting objects: 100% (600/600), done.
remote: Compressing objects: 100% (205/205), done.
remote: Total 15473 (delta 388), reused 535 (delta 358), pack-reused 14873
Receiving objects: 100% (15473/15473), 6.02 MiB | 5.70 MiB/s, done.
Resolving deltas: 100% (8265/8265), done.
ant@ripley:~/build32$ cd packaging/deb
ant@ripley:~/build32/packaging/deb$ ./build-debs.sh fixes/32
Building for fixes, v0.32 in /home/ant/build32/packaging/deb
Packaging changes between 2020-02-08 and 2022-10-19:
#checkout mythtv
if [ -d .git ]; then \
	git fetch ;\
	git checkout fixes/32 || git checkout master;\
	git pull --rebase; \
	git clean -f -d -X -e Mythbuntu/ -e mythplugins/mythweb/;\
else \
	git clone git://github.com/MythTV/mythtv.git tmp ;\
	mv tmp/.[!.]* tmp/* . ;\
	rm -rf tmp ;\
	git checkout fixes/32 || git checkout master;\
fi
Cloning into 'tmp'...
fatal: unable to connect to github.com:
github.com[0: 140.82.121.3]: errno=Connection timed out

mv: cannot stat 'tmp/.[!.]*': No such file or directory
mv: cannot stat 'tmp/*': No such file or directory
Already on 'fixes/32'
Your branch is up to date with 'origin/fixes/32'.
#checkout mythweb
if [ -d mythplugins/mythweb/.git ]; then \
	cd mythplugins/mythweb; \
	git fetch ;\
	git checkout fixes/32 || git checkout master;\
	git pull --rebase ;\
	git clean -f -d -X;\
else \
	mkdir -p mythplugins/mythweb ;\
	git clone git://github.com/MythTV/mythweb.git tmp ;\
	mv tmp/.[!.]* tmp/* mythplugins/mythweb ;\
	rm -rf tmp ;\
	cd mythplugins/mythweb ;\
	git checkout fixes/32 || git checkout master;\
fi
Cloning into 'tmp'...
^Cmake: *** [debian/mythtv.make:61: get-git-source] Interrupt

It fails for all subsequent aspects too.

Works fine for "master", but I want fixes/32.

Thanks
:)

**Comment by rcrdnalor** [comment]
Does this help when applying to fixes/32 ?
https://github.com/MythTV/packaging/commit/59f9ca09fb9acf40acdd179f3800dbe317bb265b
See
https://github.com/MythTV/packaging/issues/123

**Comment by Tinlegs** [comment]
Yes, that fixes the problem - thank you very much for your help  - You are a STAR!!!!   :)

The "mythtv.make" that arrives in the "packaging.git" for fixes/32 still has the "git" rather than the "https" reference, so I had to make the changes myself before starting the build - Granted, this wasn't a difficult thing to do, once I knew the problem, but wouldn't it be easier (and far more friendly to potential Mythtv users) to make this change at the source, rather than having to go through the process of failure?

Thanks once again for your help - Very much appreciated.  I've been a Mythtv user since about 2007 - not sure how I could now do without it LOL.
